src/components/Accordion.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import styled from 'styled-components'
+import { H3 } from './Typography';
+
+const Accordion = styled.details`
+  display: inline-block;
+  
+  & > summary:focus {
+    outline: 0;
+  }
+
+  & > summary > * {
+    display: inline-block;
+  }
+`
+
+export default ({ heading, children }) => {
+  return (
+    <Accordion>
+      <summary>
+        <H3>{heading}</H3>
+      </summary>
+      {children}
+    </Accordion>
```

Comment:
the design has the arrow on the right, do you think that violates any best-practices/a11y

Reply:
![image](https://user-images.githubusercontent.com/23178940/92044482-2f3f1600-ed33-11ea-874a-930ecc2ac8e2.png)

From what I've seen, left-side is generally best practice but I can change it to design if necessary